test/format/format.sh

```diff
@@ -199,15 +204,53 @@ success=0
 running=0
 status="format.sh-status"
 
+skip_known_errors()
+{
+	# Return if "skip_errors" is not set or -E option is not passed
+	[[ $skip_errors -ne 1 ]] && return 1
+
+	log=$1
+
+	# Define each array with multi-signature matching for a single known error
```

Comment:
Can the comment be extended to explain the 2-level concept of error vs signature that we have a list of errors to skip, and each error could have multiple issue signatures to be able to reach a finer match?

Reply:
Fixed